### ISSUE TITLE
HTTPS Certificate for bruderste.in expired

### ISSUE THREAD

**Issue by StephenWTurner** [bruderstein/nppPluginManager#129]
### Description of the Issue
v on 2018-09-23

### Steps to Reproduce the Issue
goto https://bruderste.in/npp/pm/update.htm in chrome

### Expected Behavior
webpage displayed 

### Actual Behavior
Certificate error

**Comment by chcg** [comment]
@bruderstein Could you please take care of this.

**Comment by chcg** [comment]
Certificate is update.

**Comment by nilajawill** [comment]
Hello,

I see that this issue has been closed; however, I believe this issue has re-occurred. When attempting to navigate to [https://bruderste.in/npp/pm/](https://bruderste.in/npp/pm/), Firefox refused to continue due to "the website's certificate is expired". 

_Side note: If I needed to place this note in a different place, please let me know and I will make the necessary changes._

Thank you very much!
Nilaja D Williams

**Comment by chcg** [comment]
@nilajawill You are right. But @bruderstein has to renew the certificate. Hope he will be online in the near future.

**Comment by SuperSandro2000** [comment]
Maybe use CertBot to auto renew it.

**Comment by chcg** [comment]
Should work automatically, but seems to break from time to time.

**Comment by chcg** [comment]
@bruderstein https://npppm.bruderste.in/users/login is not accessible: **502 Bad Gateway**.  Could this be fixed?
